backend/models/bug.go

```diff
@@ -0,0 +1,76 @@
+package models
+
+import (
+	"time"
+
+	"gorm.io/gorm"
+)
+
+type Bug struct {
+	ID          uint      `json:"id" gorm:"primaryKey;autoIncrement:true"`
+	UserID      string    `json:"user_id" validate:"required"`
+	ComponentID int       `json:"component_id" validate:"required"`
+	Category    string    `json:"category"`
+	Severity    string    `json:"severity"`
+	Summary     string    `json:"summary"`
+	Status      string    `json:"status"`
+	Votes       int       `json:"votes"`
+	Opened      bool      `json:"opened"`
+	OpenedAt    time.Time `json:"opened_at"`
+	UpdatedAt   time.Time `json:"updated_at"`
+}
+
+// CreateNewBug create new Bug to database
+func (d *DBClient) CreateNewBug(p Bug) (Bug, error) {
+	return p, d.Client.Create(&p).Error
+}
+
+// GetSpecificBug get bug using id from database
+func (d *DBClient) GetSpecificBug(id string) (Bug, error) {
+	p := Bug{}
+	return p, d.Client.Where("id = ?", id).First(&p).Error
+}
+
+func (d *DBClient) UpdateBug(id string, updateBug Bug) *gorm.DB {
```

Comment:
why taking the id and you have a bug struct? pass the ID to the struct before calling the function